platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/task/LearnerPushTask.java

```diff
@@ -55,6 +56,8 @@ public class LearnerPushTask<T> {
     private final LearnerTreeView<T> view;
     private final ReconnectNodeCount nodeCount;
 
+    private final ReconnectMapStats mapStats = new ReconnectMapStats() {};
```

Comment:
I do not get this. By doing so, all methods are just empty. Based on that no call to the instance has any effect. Why do you create it at all?

Reply:
This is by design in this particular PR. This is to support an iterative approach to roll out a feature with small(er), self-sufficient PRs. Please take a look at the PR description where I explain this.